msalautomationapp/src/androidTest/java/com/microsoft/identity/client/msal/automationapp/testpass/broker/crosscloud/TestCase1592468.java

```diff
@@ -137,13 +139,13 @@ public void test_acquireTokenFromCrossCloudForJoinedAccount() throws Throwable {
     }
 
     @Override
-    public LabUserQuery getLabUserQuery() {
-        final LabUserQuery query = new LabUserQuery();
-        query.userType = LabConstants.UserType.GUEST;
-        query.guestHomeAzureEnvironment = mGuestHomeAzureEnvironment;
-        query.guestHomedIn = LabConstants.GuestHomedIn.HOST_AZURE_AD;
-        query.azureEnvironment = LabConstants.AzureEnvironment.AZURE_CLOUD;
-        return query;
+    public LabQuery getLabQuery() {
+        return LabQuery.builder()
+                .userType(UserType.GUEST)
+                .guestHomeAzureEnvironment(GuestHomeAzureEnvironment.valueOf(mGuestHomeAzureEnvironment))
+                .guestHomedIn(GuestHomedIn.HOST_AZURE_AD)
+                .azureEnvironment(AzureEnvironment.AZURE_CLOUD)
+                .build();
```

Comment:
the formatting looks a bit off here

Reply:
This is how android studio auto indents it. how would you format instead?

Reply:
Looks like it is okay, it was just rendering in a weird way in GitHub: 

![image](https://user-images.githubusercontent.com/37125644/170648220-ac68beb6-6825-47dd-be6f-898212ac902f.png)